homeassistant/components/zha/entity.py

```diff
@@ -267,7 +267,11 @@ def send_removed_signal():
     @callback
     def async_state_changed_listener(self, event: Event):
         """Handle child updates."""
-        self.async_schedule_update_ha_state(True)
+        # Delay to ensure that we get updates from all members before updating the group
+        self.hass.loop.call_later(
+            0.2,
+            lambda: self.async_schedule_update_ha_state(True),
```

Comment:
Might be better to debounce it here as well.

Reply:
I didn't because async_schedule_update_ha_state() already ignores multiple calls if they happen before the state update.

Reply:
I could use the Debounce class here, but I think we will be adding complexity without much gain.

Reply:
If any attr report comes after 0.2s as a reaction to the group command, then it would trigger another round of updates. Is it the intended behavior?

Reply:
Yes, otherwise the group will be in an inconsistent state. I just want to delay the initial updates, but not ignore late ones.

Reply:
I guess it should be fine then.